### PR TITLE
[eclipse/xtext-extras#805] Release notes for incr standalone compiler

### PR DESCRIPTION
**Pull request by szarnekow**

### CODE REVIEW

File: xtext-website/_posts/releasenotes/2022-12-31-version-2-29-0.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ As you might have recognized, the number of people contributing to Xtext on a re
 
 
 ## Xtend and Java 17
-Xtext/Xtend now support Java 17 as source and target too. This requires newer Eclipse platform and JDT jars that no longer run on Java 8. Thus Xtext/Xtend 2.29 requires Java 11 as minimal Java version. While we were at we also bumped the minimal tested Eclipse version to 2022-03 (coming from Oxygen). Please Note: We did no special treatment for newer Java constructs in Xtend, so what works, works, what does not, does not. Please feel free to file bugs (and PRs) https://github.com/eclipse/xtext/issues/1982
+Xtext/Xtend now support Java 17 as source and target, too. This requires newer Eclipse platform and JDT versions that do no longer run on Java 8. That's why Xtext and Xtend 2.29 require Java 11 as minimal Java version. While we were at it we also bumped the minimal tested Eclipse version to 2022-03 (2.28 was still tested against Eclipse Oxygen). Please Note: We did no special treatment for newer Java constructs in Xtend, so what works, works, what does not, does not - the latter being most likely intentional. Please engage by filing bugs or pull requests https://github.com/eclipse/xtext/issues/1982
```
Comment (cdietrich) [inline xtext-website/_posts/releasenotes/2022-12-31-version-2-29-0.md:17]: Can you apply to Xtend rel notes too

File: xtext-website/_posts/releasenotes/2022-12-31-version-2-29-0.md [hunk 2]
```diff
@@ -28,7 +28,31 @@ Xtext/Xtend now support Java 17 as source and target too. This requires newer Ec
 
 ## Changes / Enhancements
 
-TODO Incremental Standlone Builder
+The Incremental Standlone Builder is now capable of performing incremental builds. Depending on the changes that happened since the previous compile run, only a subset of the model files need to be processed. Both model and Java file changes as well as class-path changes are taken into account. The feature is opt-in since it might break backwards compatibility for DSLs that use non-default code generation patterns. Incremental builds can also be enabled for the Xtext maven plugin. [https://github.com/eclipse/xtext-extras/issues/805](xtext-extras#805).
+
+```xml
+<plugins>
```
Comment (cdietrich) [inline xtext-website/_posts/releasenotes/2022-12-31-version-2-29-0.md:34]: https://github.com/eclipse/xtext/blob/website-master/xtext-website/documentation/350_continuous_integration.md
should this be applied here too

File: xtext-website/_posts/releasenotes/2022-12-31-version-2-29-0.md (hunk 1 above)
Comment (szarnekow) [reply xtext-website/_posts/releasenotes/2022-12-31-version-2-29-0.md]: Yes

File: xtext-website/_posts/releasenotes/2022-12-31-version-2-29-0.md (hunk 2 above)
Comment (szarnekow) [reply xtext-website/_posts/releasenotes/2022-12-31-version-2-29-0.md:34]: Yes, done.

**Review by cdietrich — APPROVED** [review]

**Pull request merged**